src/test/java/com/ericsson/ei/testsuite/SuiteTest.java

```diff
@@ -0,0 +1,57 @@
+package com.ericsson.ei.testsuite;
```

Comment:
Should this file be here?

Reply:
the package testsuite is not needed and deleted